app/services/preserved_object_handler.rb

```diff
@@ -73,11 +73,16 @@ def check_existence
 
           raise_rollback_if_pc_po_version_mismatch(pres_copy.version, pres_object.current_version)
 
+          incoming_vers_eq_pc_vers = (incoming_version == pres_copy.version)
```

Comment:
i do have a slight preference for the conditions in the if statement, but i dont mind it here.